notebooker/web/utils.py

```diff
@@ -39,6 +39,8 @@ def get_all_possible_templates(warn_on_local=True):
         from notebooker import notebook_templates_example
 
         all_checks = get_directory_structure(os.path.abspath(notebook_templates_example.__path__[0]))
+    if subfolder is not None:
+        all_checks = {c: val for c, val in all_checks.items() if c.startswith(subfolder)}
```

Comment:
I think this has the unintended side effect of removing the possibility to execute notebooks via "Execute a Notebook" dialog - see:

![image](https://github.com/man-group/notebooker/assets/3275431/be750ee6-9fdd-45e7-98ae-ed96b8f5ba0f)
![image](https://github.com/man-group/notebooker/assets/3275431/047aed53-11aa-4607-8536-d59d5c974dc7)

In the case of "leaf" folders, it hides everything. We need to do this filtering elsewhere.